src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
 spring.application.name=apiPolicy
+url_wiremock=http://localhost:8081
```

Comment:
I think we already talked about naming before. What's wiremock to your API? The service or a tool to mock the service? Also this isn't following standard practice either look at the previous property, do they look alike? Spring is namespacing its properties (spring.) and dividing hierarchy with a name convention (spring.application.)... when in rome... you know the saying...